src/app/credExplorer/weights/weights.js

```diff
@@ -14,8 +15,8 @@ export type WeightedEdgeType = {|
 |};
 
 export type WeightedTypes = {|
-  +nodes: $ReadOnlyArray<WeightedNodeType>,
-  +edges: $ReadOnlyArray<WeightedEdgeType>,
+  +nodes: Map<NodeAddressT, WeightedNodeType>,
```

Comment:
Should document what these addresses mean: it is not obvious that the
key is required to map to the prefix of the type of its corresponding
value.